data/categories.txt

```diff
@@ -2850,7 +2850,7 @@ highway-primary|highway-primary_link|highway-residential|highway-secondary|highw
 en:street|st|str|ave|road|rd
 ru:улица|переулок|проезд|тракт|проспект|ул
 uk:вулиця|провулок|проїзд|проспект|бульвар|вул
-de:Strasse|Weg|Avenue|Gasse|Str
+de:Straße|Weg|Boulevard|Gasse
```

Comment:
I thought that searching for Str would match on **Str**aße as well. Is that so?  
Does the program translate "ß" to "ss" and similar stuff? Else I would have to add back "Strasse" … which is just the way of writing "Straße" when people have no "ß" at hand (that is nearly everybody outside Austria/Germany).